src/sbt-test/sbt-1.5/scala-3/build.sbt

```diff
@@ -0,0 +1,6 @@
+inThisBuild(
+  Seq(
+    scalaVersion := "3.0.0-RC3",
+    scalafixDependencies +="ch.epfl.scala" %% "example-scalafix-rule" % "1.6.0",
```

Comment:
this contains very simple syntactic and semantic rules so it's good for an initial scripted until built-in rules pick up actual support